dist/README.md

```diff
@@ -1,59 +1,36 @@
+Start by creating a new branch for a new release:
 ```
+git checkout -b iiris-release-4.2.8-2-branch iiris-release-4.2.8-1-branch
+git push -u origin iiris-release-4.2.8-2-branch
+```
+Create a new feature branch for your task, e.g.
+```
+git checkout -b feature/IISCRUM-2796 iiris-release-4.2.8-2-branch
+git push -u origin feature/IISCRUM-2796
+```
+Make your changes and create a pull request in GitHub and when that is merged you can create a new release.
 
-Or see more installation options in [docs](https://alexanderzobnin.github.io/grafana-zabbix/installation/).
-
-## Getting started
-
-First, [configure](https://alexanderzobnin.github.io/grafana-zabbix/configuration/) Zabbix data source. Then you can create your first dashboard with step-by-step [Getting started guide](https://alexanderzobnin.github.io/grafana-zabbix/guides/gettingstarted/).
-
-## Documentation
-
-- [About](https://alexanderzobnin.github.io/grafana-zabbix)
-- [Installation](https://alexanderzobnin.github.io/grafana-zabbix/installation)
-- [Getting Started](https://alexanderzobnin.github.io/grafana-zabbix/guides/gettingstarted)
-- [Templating](https://alexanderzobnin.github.io/grafana-zabbix/guides/templating)
-- [Alerting](https://alexanderzobnin.github.io/grafana-zabbix/reference/alerting/)
-- [Metric processing functions](https://alexanderzobnin.github.io/grafana-zabbix/reference/functions/)
-
-## Community Resources, Feedback, and Support
+To build grafana-zabbix panel, go to it's folder and run:
+`make dist`
 
-- Found a bug? Want a new feature? Feel free to open an [issue](https://github.com/alexanderzobnin/grafana-zabbix/issues/new).
-- Have a question? You also can open issue, but for questions, it would be better to use [Grafana Community](https://community.grafana.com/) portal.
-- Need additional support? Contact me for details [alexanderzobnin@gmail.com](mailto:alexanderzobnin@gmail.com)
+Compiled version is in dist-folder and that is included in grafana-zabbix repo because Grafana's Docker build will fetch the compiled version from there.
+Before releasing you need to sign the plugin with instructions below and copy to MANIFEST.txt to Iiris-repos Grafana-folder where Dockerfile is located.
+Create a new tag by leaving the 'branch' postfix away.
+`iiris-release-4.2.8-2`
 
----
-:copyright: 2015-2022 Alexander Zobnin alexanderzobnin@gmail.com
+## What is MANIFEST.txt
+MANIFEST.txt is the signature file iiris-grafana-zabbix panel or any Grafana plugin. Manifest is created to dist-folder during sign process.
+If iiris-grafana-zabbix panel is updated, you need to re-create the MANIFEST.txt and copy the new file to Grafana-folder where Docker file is located.
 
-Licensed under the Apache 2.0 License
+## Signing grafana-zabbix plugin
+Go to the iiris-grafana-zabbix folder and run:
+```
+make dist
+export GRAFANA_API_KEY=eyJrIjoiNDAzMTY3Y2I2YjNjYWVkZjIyY2VmNWVlYmQxOTVhYTA4YWYxZWI5ZiIsIm4iOiJ0aW15bGx5bV9hcGlfa2V5IiwiaWQiOjMxNDQ5fQ==
```

Comment:
Remove the GRAFANA_API_KEY and npx part from this file since this is public repo. They need to be removed also from iiris-release-4.2.8-2-branch where I added it without thinking. You should probably change the api key in future because it is done with my digia email address.

Reply:
In commits 52d68fec99a85c88cd549cef8f471cb27e67fbf9 and aace964ab5890a22403ba8ca07b2d99f210ba389.